src/main/kotlin/org/jitsi/jicofo/JicofoServices.kt

```diff
@@ -94,20 +94,24 @@ open class JicofoServices(
             }
         }
 
-        val focusJid = XmppConfig.client.username.toString() + "@" + XmppConfig.client.domain.toString()
         conferenceRequestHandler = ConferenceRequestHandler(
             focusManager = focusManager,
-            focusAuthJid = focusJid,
+            focusAuthJid = XmppConfig.client.username.toString() + "@" + XmppConfig.client.domain.toString(),
             isFocusAnonymous = StringUtils.isBlank(XmppConfig.client.password),
             authAuthority = authenticationAuthority,
             reservationSystem = reservationSystem
         )
-        focusComponent = FocusComponent(XmppComponentConfig.config, conferenceRequestHandler).apply {
-            val configService = ServiceUtils2.getService(bundleContext, ConfigurationService::class.java)
-            loadConfig(configService, "org.jitsi.jicofo")
-            authenticationAuthority?.let { setAuthAuthority(authenticationAuthority) }
+
+        focusComponent = if (XmppComponentConfig.config.enabled) {
+            FocusComponent(XmppComponentConfig.config, conferenceRequestHandler).apply {
+                val configService = ServiceUtils2.getService(bundleContext, ConfigurationService::class.java)
+                loadConfig(configService, "org.jitsi.jicofo")
+                authenticationAuthority?.let { setAuthAuthority(authenticationAuthority) }
```

Comment:
nit: I don't know if this was intentional or not (I've done this same thing before on accident), but I think it's more idiomatic to use `it` here:

```
authenticationAuthority?.let { setAuthAuthority(it) }
```

Using `authenticationAuthority` works since it's a val and it gets smart-cast, but I think it's a little more brittle to use `authenticationAuthority` rather than the explicit result of the `let`.

Reply:
From what I saw this got fixed in a later commit